plugins/package-managers/node/src/funTest/kotlin/NpmFunTest.kt

```diff
@@ -40,7 +40,8 @@ import org.ossreviewtoolkit.utils.test.patchExpectedResult
 
 class NpmFunTest : WordSpec({
     "NPM" should {
-        "resolve shrinkwrap dependencies correctly" {
+        "resolve dependencies for a project with a 'shrinkwrap.json' correctly" {
+            // See also: https://docs.npmjs.com/cli/v6/configuring-npm/shrinkwrap-json.
```

Comment:
Actually, I'm not sure that the comment is useful here. In what regard should I look at the description of a `npm-shrinkwrap.json` in the context of this test?

IMO such a link would be more useful in the code under test that deals with the `npm-shrinkwrap.json` file, like the `NodePackageManager` enum.

TL;DR I'd just remove the `README.md`, and optionally document the enums in `NodePackageManager` in another commit with references to the mentioned files.

And while at it a commit title nit: `README.md` could be put in backticks.